external/netfx/netfx.depproj

```diff
@@ -48,7 +42,8 @@
 
   <ItemGroup>
     <PackageReference Include="$(TargetingPackNugetPackageId)">
-      <Version>1.0.1</Version>
+      <Version Condition="'$(TargetingPackNugetPackageId)' == 'Microsoft.TargetingPack.NETFramework.v4.7.1'">1.0.0-preview</Version>
```

Comment:
We should probably re ship a targeting pack without the preview tag, just to prevent from it getting cleaned out by mistake.